sudoers_file/prometheus

```diff
@@ -0,0 +1 @@
+prometheus ALL=(ALL) NOPASSWD: /usr/sbin/lnetctl
```

Comment:
I think this will be more specific:
```suggestion
prometheus ALL=(ALL) NOPASSWD: /usr/sbin/lnetctl stats show
prometheus ALL=(ALL) NOPASSWD: /usr/sbin/lnetctl net show -v 4
```